molo/core/models.py

```diff
@@ -556,13 +556,57 @@ class LanguageRelation(models.Model):
     language = models.ForeignKey('core.SiteLanguage', related_name='+')
 
 
+def get_translation_for_new_qs(qs, locale, site, is_live=True):
```

Comment:
should I add this to my PR @KaitCrawford for molo.core translation fields.

Reply:
@sewagodimo no. this part of the code is for the ticket I'm working on. Your ticket is for adding the fields to all the models that inherit from TranslatablePageMixin